lib/graphql/account.ts

```diff
@@ -13,10 +13,31 @@ const GET_ACCOUNT = `query {
     }
   }
 }`;
+const GET_ACCOUNT_STATS = `query {
+  viewer {
+    mainAccount {
+      accountStats {
```

Comment:
so it should now be `stats` here too, right? 😛

Reply:
dont check the changes too fast! 😅  You dont give me the chance to hide my mistakes  😜